src/statmodels.jl

```diff
@@ -79,20 +108,48 @@ when applicable the intercept and the distribution's dispersion parameter.
 """
 dof(obj::StatisticalModel) = error("dof is not defined for $(typeof(obj)).")
 
+"""
+    ess(obj::StatisticalModel)
+
+Return the the explained sum of squares.
+"""
+ess(obj::StatisticalModel) = sum(abs2, fitted(obj) - meanresponse(obj))
+
+"""
+    rss(obj::StatisticalModel)
+
+Return the the residual sum of squares.
+"""
+rss(obj::StatisticalModel) = sum(abs2, (model_response(obj) - fitted(obj)))
+
 """
     stderr(obj::StatisticalModel)
 
 Return the standard errors for the coefficients of the model.
 """
 stderr(obj::StatisticalModel) = sqrt.(diag(vcov(obj)))
 
+"""
+    tss(obj::StatisticalModel)
+
+Return the standard errors for the coefficients of the model.
```

Comment:
Should this be the total sum of squares?

Reply:
I figured a good amount of methods will be dependent on their types, but leaving those with the error methods has no drawbacks that I can think of.
Thanks for catching that.